webpack.config.js

```diff
@@ -51,6 +51,7 @@ module.exports = {
     new webpack.ProvidePlugin({
       'fetch': 'imports?this=>global!exports?global.fetch!whatwg-fetch'
     }),
+    new webpack.ContextReplacementPlugin(/buffer/, require('buffer')),
```

Comment:
yeah this is broken if you do npm install for webpack with latest version, so this is just workaround for getting webpack with this dependency to work for now until they patch this upstream.